stdlib/http/src/main/java/org/ballerinalang/net/http/HttpDispatcher.java

```diff
@@ -66,9 +64,7 @@
  */
 public class HttpDispatcher {
 
-    private static final Logger breLog = LoggerFactory.getLogger(HttpDispatcher.class);
-
-    protected static HttpService findService(HTTPServicesRegistry servicesRegistry, HttpCarbonMessage inboundReqMsg) {
+    public static HttpService findService(HTTPServicesRegistry servicesRegistry, HttpCarbonMessage inboundReqMsg) {
```

Comment:
Why have we changed the visibility of this method?

Reply:
Because all the methods in this class are static and `WebSubDispatcher` has extended the class only to use this method. There is no harm in having this public right? Then we can have a private constructor for the class.

Reply:
+1